history_tool.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from collections.abc import AsyncIterable
+
+class History:
+
+    def __init__(self, message :str, date=datetime.now()) -> None:
+        self.message = message
+        self.date = date
+
+    def __lt__(self, other) -> bool:
```

Comment:
oh i dont like override special methods because we now need a documentation to describe what this doing (because dont have a name)

Reply:
+ you sometimes checks datetime or message - is a little confused